kubelet-to-gcm/monitor/kubelet/translate.go

```diff
@@ -223,20 +303,17 @@ func (t *Translator) translateContainers(pods []stats.PodStats) ([]*v3.TimeSerie
 
 func (t *Translator) translateContainer(podID, namespace string, container stats.ContainerStats, requireFsStats bool) ([]*v3.TimeSeries, error) {
 	var (
-		containerSeries []*v3.TimeSeries
+		containerSeries, memTS, rootfsTS, logfsTS, cpuTS []*v3.TimeSeries
+		err error
 		containerName   = container.Name
+		containerLabels = map[string]string{
+			"namespace": 	namespace,
+			"pod":			podID,
```

Comment:
nit: On GitHub the whitespaces here seems misaligned, may need running gofmt.

Reply:
Done.

Reply:
It seems this still file needs some reformatting.